src/transport/msg_schemas.py

```diff
@@ -145,6 +146,7 @@ def check_pause_message(msg: PauseMessage) -> bool:
 
         if not verified:
             logger.error({'msg': 'Message verification failed.', 'value': msg})
+            UNEXPECTED_EXCEPTIONS.labels('pause_message_verification_failed').inc()
```

Comment:
may be we can add `module_id` as a label. it would be good to separate this metric by the module id on the dashboard and have this info in alerts

Reply:
Not sure this info is usefule. I see runbook flow like
- Saw alert at 10.15 PM
- Go to logs at 10.15 PM
- See error log and check why it was boken